po/de.po

```diff
@@ -525,7 +525,7 @@ msgstr "Erkennen deaktiviert"
 #. TRANSLATORS: Open a dialog to connect to an IP or Bluez device
 #: data/ui/preferences-window.ui:821
 msgid "Add device by IP…"
-msgstr ""
+msgstr "Gerät nach IP hinzufügen …"
```

Comment:
Questioning the space before that ellipsis, but since I don't speak German, what do I know? Let's go with this.

Reply:
> Questioning the space before that ellipsis, but since I don't speak German, what do I know? Let's go with this.

It is intentional. In German, the ellipsis (triple points) should have a space before it.

You can test the German text on LanguageTool. Try to delete the space before the ellipsis and see what LanguageTool suggests.
https://languagetool.org

Reply:
@Amr-Ibra 

Ooh, LanguageTool is a great resource, thanks for that!

I did toss it through Google Translate (I _knowwwwww_, believe me), and it (both unhelpfully and, sadly, unsurprisingly) was more than happy to translate it both ways.

With the un-spaced version as the default, even. (Though, that may be down to it being a smaller diff from the source string. Who knows how its "mind" works?)